lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/fs.rb

```diff
@@ -20,6 +21,7 @@ class Console::CommandDispatcher::Stdapi::Fs
 
   include Console::CommandDispatcher
   include Rex::Post::Meterpreter::Extensions::Stdapi
+  include Rex::Post::LocalFs
```

Comment:
Let's remove all of the copy pasta:


```
# Current location -> new location

# Reusable command dispatcher (for postgres, mssql, smb etc)
lib/rex/post/smb/ui/console/command_dispatcher/local_fs.rb -> lib/msf/ui/console/command_dispatcher/local_file_system.rb

# Module mixin
lib/rex/post/local_fs.rb -> lib/msf/ui/console/local_filesystem.rb
```

And then just update the enstack calls with the new module namespace:

```
enstack_dispatcher(Msf::Ui::Console::CommandDispatcher::LocalFileSystem)
```

And remember to update Meterpreter's `include` call to the new reference too :+1: